eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/NewEpoch.hs

```diff
@@ -166,6 +169,8 @@ newEpochTransition = do
             -- map is empty (db-sync depends on it).
             tellEvent $ TotalRewardEvent e regRU
             pure es'
+      let adaPots = totalAdaPotsES es
```

Comment:
As discussed in https://github.com/input-output-hk/cardano-ledger/pull/2791#discussion_r874619228 I think `es'''` pots better fits what we need.

Reply:
Yes! Thank you for all the explanation, I'm just about to push a new version with this. 
Thanks!